--- src/Makefile.am.link	2011-05-09 12:08:24.000000000 +0200
+++ src/Makefile.am	2011-05-09 12:08:48.000000000 +0200
@@ -76,6 +76,8 @@
 LDADD        = $(top_builddir)/zlib/src/libz.la
 endif
 
+libid3_la_LIBADD = -lstdc++ -lz
+
 libid3_la_LDFLAGS = \
   -version-info $(LT_VERSION) \
   -release $(LT_RELEASE) \
--- src/Makefile.in.link	2011-05-09 12:08:55.000000000 +0200
+++ src/Makefile.in	2011-05-09 12:09:14.000000000 +0200
@@ -192,7 +192,7 @@
 CONFIG_CLEAN_FILES =
 LTLIBRARIES = $(lib_LTLIBRARIES)
 
-libid3_la_LIBADD =
+libid3_la_LIBADD = -lstdc++ -lz
 am__objects_1 = c_wrapper.lo field.lo field_binary.lo field_integer.lo \
 	field_string_ascii.lo field_string_unicode.lo frame.lo \
 	frame_impl.lo frame_parse.lo frame_render.lo globals.lo \
